test/replicache_test.dart

```diff
@@ -38,14 +78,38 @@ Future<void> main() async {
 
   // TODO(arv): Start the test server from here!
 
+  List replays = [];
+  String recordPath;
+
+  Future<void> useReplay(String path) async {
+    final replaysString = await File(path).readAsString();
+    replays = json
+        .decode(replaysString)
+        .toList()
+        .map((data) => Replay.fromJson(data))
+        .toList();
+  }
+
   const MethodChannel(CHANNEL_NAME)
       .setMockMethodCallHandler((methodCall) async {
     final method = methodCall.method;
     final String dbName = methodCall.arguments[0];
     final Uint8List data = methodCall.arguments[1];
+
+    if (recordPath == null && replays.isNotEmpty) {
+      final i = replays.indexWhere((r) => r.matches(dbName, method, data));
```

Comment:
[taste] Is it possible to just play the responses in order, one after the next, instead of searching for a match?

Reply:
Not at the moment since 'pull' comes along at "random" times. I'm tempted to expose `Replicache.forTesting` which would allow disabling sync and access `maybeEndSync` etc.

Reply:
I think it would be useful to disable sync anyway, so you could make that a feature. `Replicache.syncInterval=null`

Reply:
Another reason to have forTesting is to allow awaiting for `_opened`.

Reply:
Still. I agree. We should allow pausing the sync interval